cf/dynamo.yaml

```diff
@@ -45,7 +45,31 @@ Resources:
       AttributeDefinitions:
         - AttributeName: "experimentId"
           AttributeType: "S"
+        - AttributeName: "projectUuid"
+          AttributeType: "S"
       KeySchema:
         - AttributeName: "experimentId"
           KeyType: "HASH"
+      GlobalSecondaryIndexes:
+        - IndexName: gsiExperimentid
```

Comment:
choose a more descriptive name for this

Reply:
changed to `gsiByProjectAndExperimentID`